onboarding/app/controllers/customer_controller.rb

```diff
@@ -0,0 +1,29 @@
+class CustomerController < ApplicationController
+  before_action :require_active_customer
+
+  def create
+    @customer = Customer.new(params[:customer])
```

Comment:
You need to whitelist the params in order to use it this way.

Reply:
You can implement a before action that permits all the used keys in the controller.